project/Deps.scala

```diff
@@ -6,20 +6,20 @@ object Resolvers {
 
 object Deps {
   object V {
-    val algebird = "0.12.2"
+    val algebird = "0.13.0"
     val jackson = "1.9.13"
-    val bijection = "0.7.0"
     val bonsai = "0.2.1"
+    val bijection = "0.9.5"
     val tDigest = "3.1"
 
     val hadoopClient = "2.5.2"
     val scalding = "0.16.1-RC3"
-    val chill = "0.5.2"
+    val chill = "0.9.2"
```

Comment:
no. This same argument doesn't apply. For a long time we have been on `2.x` of Kryo, all versions <= 0.7 are on that version (all in the last 3 years are 2.21). 0.8.x is on Kryo 3, 0.9 is on Kryo 4.

Basically, Kryo didn't change much for a long time, but then it got a bit more regular and disciplined with releases.